meter-health/meter_health.py

```diff
@@ -1,98 +0,0 @@
-import requests
```

Comment:
Try to use Pep8 recommendations  in this section
[https://www.python.org/dev/peps/pep-0008/#imports](url)